sources/procfs.go

```diff
@@ -239,13 +239,13 @@ func (s *lustreSource) generateMDTMetricTemplates() error {
 func (s *lustreSource) generateMGSMetricTemplates() error {
 	metricMap := map[string][]lustreHelpStruct{
 		"mgs/MGS/osd/": {
-			{"blocksize", "blocksize", "Filesystem block size in bytes"},
-			{"filesfree", "filesfree", "The number of inodes (objects) available"},
-			{"filestotal", "filestotal", "The maximum number of inodes (objects) the filesystem can hold"},
-			{"kbytesavail", "kbytesavail", "Number of kilobytes readily available in the pool"},
-			{"kbytesfree", "kbytesfree", "Number of kilobytes allocated to the pool"},
-			{"kbytestotal", "kbytestotal", "Capacity of the pool in kilobytes"},
-			{"quota_iused_estimate", "quota_iused_estimate", "Returns '1' if a valid address is returned within the pool, referencing whether free space can be allocated"},
+			{"blocksize", "mgs_blocksize_bytes", "Filesystem block size in bytes"},
+			{"filesfree", "mgs_files_free_now_total", "The number of inodes (objects) available"},
```

Comment:
Same as with the OST, should this just be `mgs_files_free_now_bytes` since this is the current value and not a counter?